service/src/main/java/gov/nasa/pds/api/registry/search/HitIterator.java

```diff
@@ -12,12 +12,12 @@
 
 public class HitIterator implements Iterable<Map<String,Object>>,Iterator<Map<String,Object>>
 {
-	private int size=10; // define size to use here to prevent page skipping if opensearch default size ever changes
+	private int size=500; // define size to use here to prevent page skipping if opensearch default size ever changes
```

Comment:
So this is all it takes? Just bump this up? If OpenSearch changes again will we need to change this again?

Reply:
Negative(ish) - this is primarily a factor of data volume.  Given `HitIterator.size=n` and some OpenSearch internal paging config value `m` (currently set to 500 and hidden deep enough that I doubt it's likely to change unless something breaks), HitIterator will make a somewhat-expensive request for every `n*m` products it needs to fetch.

If the thrust is "Hm, seems like this should be a config value bro" then yes, yes it should (and should be extracted at $laterDate), but I'm changing it here for expediency's sake.